packages/appkit-utils/src/bitcoin/BitcoinTypesUtil.ts

```diff
@@ -69,6 +69,13 @@ export namespace BitcoinConnector {
     recipient: string
   }
 
+  /**
+   * Parameters for signing a PSBT. The names or types of these parameters may vary across different providers. Refer to the links below for more information:
+   * @link https://docs.unisat.io/dev/open-api-documentation/unisat-wallet#signpsbt
+   * @link https://web3.okx.com/build/dev-docs/sdks/chains/bitcoin/provider#signpsbt
+   * @link https://developer.onekey.so/connect-to-software/webapp-connect-onekey/btc/api-reference/signpsbt
+   * @link https://leather.gitbook.io/developers/bitcoin-methods/signpsbt
```

Comment:
shouldn't we have a unified interface for devs and map internally to corresponding actions in wallets?